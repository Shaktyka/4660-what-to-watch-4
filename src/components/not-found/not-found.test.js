import React from 'react';
import renderer from 'react-test-renderer';

import NotFound from './not-found.jsx';

describe(`NotFound rendering`, () => {

  it(`NotFound renders correctly`, () => {
    const tree = renderer
      .create(
          <NotFound />, {
            createNodeMock: () => {
              return {};
            }
          })
      .toJSON();

    expect(tree).toMatchSnapshot();
  });

});
