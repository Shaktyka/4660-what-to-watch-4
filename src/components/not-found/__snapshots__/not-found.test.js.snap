// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound rendering NotFound renders correctly 1`] = `
<div
  className="user-page"
  style={
    Object {
      "textAlign": "center",
    }
  }
>
  <header
    className="page-header user-page__head"
  >
    <div
      className="logo"
    >
      <a
        className="logo__link"
        href="/"
        onClick={[Function]}
      >
        <span
          className="logo__letter logo__letter--1"
        >
          W
        </span>
        <span
          className="logo__letter logo__letter--2"
        >
          T
        </span>
        <span
          className="logo__letter logo__letter--3"
        >
          W
        </span>
      </a>
    </div>
    <h1
      className="page-title user-page__title"
    >
      404
    </h1>
  </header>
  <div
    className="sign-in user-page__content"
  >
    <p
      style={
        Object {
          "fontSize": "30px",
        }
      }
    >
      Page not found
    </p>
    <a
      href="/"
      onClick={[Function]}
      style={
        Object {
          "color": "#dfcf77",
          "fontWeight": 700,
          "textDecoration": "none",
        }
      }
    >
      Go to main page
    </a>
  </div>
  <footer
    className="page-footer"
  >
    <div
      className="logo"
    >
      <a
        className="logo__link logo__link--light"
        href="/"
        onClick={[Function]}
      >
        <span
          className="logo__letter logo__letter--1"
        >
          W
        </span>
        <span
          className="logo__letter logo__letter--2"
        >
          T
        </span>
        <span
          className="logo__letter logo__letter--3"
        >
          W
        </span>
      </a>
    </div>
    <div
      className="copyright"
    >
      <p>
        Â© 2019 What to watch Ltd.
      </p>
    </div>
  </footer>
</div>
`;
